docs/appendices/release-notes/unreleased.rst

```diff
@@ -87,6 +87,13 @@ None
 
 Changes
 =======
+- Added support for ``CAST(<INTERVAL> AS LONG)`` to get the total number of
+  milliseconds in an interval, e.g.::
+
+    SELECT (ts_end - ts_start)::long FROM test
+    SELECT (ts_end - ts_start)::long / (1000 * 60 * 60 * 24) FROM test
```

Comment:
Isn't this the same as `extract (epoch from (ts_end - ts_end))` ?

Reply:
we didn't add support for `epoch` as `epoch` refers to a timestamp since `1/1/1970 00:00.000`, and it didn't seem a valid usage, that's why I decided to implement the cast.